src/lib/markets/index.ts

```diff
@@ -714,6 +720,20 @@ export class MarketsModule {
 
         return buildIntentSigningPayload({ chainId: this.config.chainId, ...args, address })
       },
+
+      relayedPlaceOrder: (args: OmitBound<BuildPlaceOrderSigningPayloadArgs> & OptionalAddress) => {
```

Comment:
Should this just be placeOrder since it is "relayed" but not in the same sense that the other "relayed" types have inner / outer signatures

Reply:
I guess its more dependent on the action structure rather than inner/outer payloads

Reply:
Yeah it gets a bit messy because we already have a `placeOrder` which is sending the actual transaction...

Maybe `signedPlaceOrder` is cleanest.